solutions/standard/variables.tf

```diff
@@ -209,6 +209,65 @@ variable "auto_scaling" {
   default     = null
 }
 
+##############################################################################
+## Secrets Manager Service Credentials
+##############################################################################
+
+variable "skip_rd_sm_auth_policy" {
```

Comment:
I suggest changing this to `skip_redis_sm_auth_policy`. Ive never seen redis shortened to `rd`